src/runtime/engines/singularity/container.go

```diff
@@ -499,7 +499,6 @@ func (c *container) addRootfsMount(system *mount.System) error {
 		if err := system.Points.AddBind(mount.RootfsTag, rootfs, c.session.RootFsPath(), flags); err != nil {
 			return err
 		}
-		system.Points.AddRemount(mount.RootfsTag, c.session.RootFsPath(), flags)
```

Comment:
Instead of removing `AddRemount` it's better to add a check `if !c.userNS { AddRemount... }` to force remount with setuid workflow and correctly apply `MS_NOSUID | MS_NODEV ` option

Reply:
Oh, so that's what the remount is for.  Ok, I will update for that change.